src/main/java/com/buddy/api/domains/profile/repositories/ProfileRepository.java

```diff
@@ -1,10 +1,14 @@
 package com.buddy.api.domains.profile.repositories;
 
+import com.buddy.api.domains.account.entities.AccountEntity;
 import com.buddy.api.domains.profile.entities.ProfileEntity;
+import java.util.Optional;
 import java.util.UUID;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface ProfileRepository extends JpaRepository<ProfileEntity, UUID> {
+
+    Optional<ProfileEntity> findByAccount(AccountEntity account);
```

Comment:
Aqui está errado. O relacionamento é de 1 `ACCOUNT` para 1 ou 0 ou mais `PROFILE`, assim, deveria ser:
```suggestion
    List<ProfileEntity> findByAccount(AccountEntity account);
```

Reply:
Agora notei que esse método não é usado em nenhum momento, assim, por qual motivo criar ele agora?

Reply:
necessário para o teste de integração na classe: src/test/java/com/buddy/api/integrations/web/profile/controller/CreateProfileControllerTest.java

Reply:
Dado que retorna varios perfils pra uma conta, então vai ser necessário testar isso também.